### ISSUE TITLE
Package External References not written by jsonyamlxml writer

### ISSUE THREAD

**Issue by jasinner** [spdx/tools-python#249]
When a document has packages with package external references those are not written by the jsonyamlxml writer. 

Compare that behaviour with the [tagvaluewriter](https://github.com/spdx/tools-python/blob/2f4225c5a2eb2d1ee577c53b3b6075c91b0981c1/spdx/writers/tagvalue.py#L272).

**Comment by meretp** [comment]
Thanks for the issue! Looking deeper into this issue I found out that the jsonyamlxml-parser and builder are also missing ability to read external references for packages. It might be good to fix this as a whole (parser, builder and writer) in one PR. As discussed in #244 we are planning to refactor the builder. Once that is done we can also address this issue. Are you okay with this issue being put on hold or is it urgent, @jasinner?

**Comment by nicoweidner** [comment]
Should be resolved via https://github.com/spdx/tools-python/pull/266, please give it another go, then we can close this issue
